dev/merge_arrow_pr.py

```diff
@@ -109,15 +110,10 @@ def strip_ci_directives(commit_message):
     # commit message
     return _REGEX_CI_DIRECTIVE.sub('', commit_message)
 
-
-def fix_version_from_branch(branch, versions):
+def fix_version_from_branch(versions):
```

Comment:
There's a minor linting issue missing an extra empty line here:
```suggestion

def fix_version_from_branch(versions):
```